docs/guide/terraform-guide.md

```diff
@@ -0,0 +1,181 @@
+- [Apache](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_apache)
+- [Cassandra](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_cassandra)
+- [ClickHouse](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_clickhouse)
+- [Docker](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_docker)
+- [Elasticsearch](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_elasticsearch)
+- [HAProxy](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_haproxy)
+- [HBase](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_hbase)
+- [Java](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_jvm)
+- [Kafka](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_kafka)
+- [Logs](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_logsene)
+- [MongoDB](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_mongodb)
+- [MySQL](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_mysql)
+- [Nginx](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_nginx)
+- [Nginx+](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_nginxplus)
+- [Node.js](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_nodejs)
+- [Redis](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_redis)
+- [Hadoop](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_hadoopmrv1)
+- [Solr](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_solr)
+- [Solr Cloud](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_solrcloud)
+- [Apache Spark](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_spark)
+- [Apache Storm](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_storm)
+- [Tomcat](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_tomcat)
+- [ZooKeeper](https://registry.terraform.io/providers/sematext/sematext/latest/docs/resources/sematext_monitor_zookeeper)
+
+
+<br>
+
+
+## Getting Started
+
+
+### Setup
+
+You'll need the following : 
+
+- [Terraform v0.13+](https://www.terraform.io/downloads.html) 
+- [Sematext Cloud Account](https://apps.sematext.com/ui/account/api)
```

Comment:
Should that point to account/api or just account?

Reply:
Done